docs/source/guides/parallel.rst

```diff
@@ -62,3 +62,11 @@ The dashboard requires an additional python package, bokeh, to work. Once bokeh
 Parallel Computation by Partitioning Data
 -----------------------------------------
 As an alternative to Featuretool's parallelization, the data can be partitioned and the feature calculations run on multiple cores or a cluster using Dask or Apache Spark with PySpark. This approach may be necessary with a large ``EntitySet`` because the current parallel implementation sends the entire ``EntitySet`` to each worker which may exhaust the worker memory. For more information on partitioning the data and using Dask or Spark, see :doc:`/guides/performance`. Dask and Spark allow Featuretools to scale to multiple cores on a single machine or multiple machines on a cluster.
+
+Computation with a Dask EntitySet (BETA)
```

Comment:
i'd move this section up to the top and we need to also edit the "Running Featuretools with Spark and Dask" section to align with this

Reply:
Want to make sure I fully understand this comment...

So, you want this new Dask section to be the very first section in the document instead of the last?

Can you provide a little more info on what you are thinking in regards to aligning the "Running Featuretools with Spark and Dask" section with this?

Reply:
> So, you want this new Dask section to be the very first section in the document instead of the last?

Yep

> Can you provide a little more info on what you are thinking in regards to aligning the "Running Featuretools with Spark and Dask" section with this?

Right now that section says "If you have a big data problem and are interested in testing our latest Dask or Spark integrations for free, please let us know by completing this simple request form." However, if they want to use dask, they dont need to fill out the form since dask is now released. They should still fill out the form for spark though

Reply:
@kmax12 If we remove the reference to Dask in this section (and the corresponding section in `Improving Computational Performance` - we also need to update the linked request form which still mentions Dask. Can you update that?

Reply:
yep, ill do that right after we release! thanks for the reminder